client/src/declarations.d.ts

```diff
@@ -1,2 +1,5 @@
 // eslint-disable-next-line import/unambiguous
 declare module '@freecodecamp/react-bootstrap';
+declare module 'redux-actions';
+declare module 'react-redux';
+declare module '@loadable/component';
```

Comment:
I appreciate that I set a bad precedent here, but can we use type declarations where we have them?

https://www.npmjs.com/package/@types/redux-actions
https://www.npmjs.com/package/@types/react-redux
https://www.npmjs.com/package/@types/loadable__component

should do the trick.

Reply:
Are you saying we should be installing and importing those instead?

Reply:
Yep.  I think that's necessary for TypeScript to understand those packages, since they don't provide their own types.